numberdrome.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+class Numberdrome {
+  constructor() {
+    this.numbersArr = [];
+  }
+
+  addNumber(n) {
+    this.numbersArr.push(n);
+  }
+
+  removeNumber(n) {
+    this.numbersArr.splice(this.numbersArr.indexOf(n), 1);
+  }
+
+  sum() {
+    return this.numbersArr.length > 0 ? this.numbersArr.reduce((a,b) => a + b, 0) : 0;
```

Comment:
unneeded ternary, reduce will return 0 on the empty array
add space after comma